app/src/main/AndroidManifest.xml

```diff
@@ -51,8 +51,21 @@
       android:name=".app.help.HelpActivity"
       android:label="@string/help_activity_title"
       android:theme="@style/OppiaThemeWithoutActionBar" />
+    <activity
+      android:name=".app.help.thirdparty.LicenseListActivity"
+      android:label="@string/license_list_activity_title"
+      android:theme="@style/OppiaThemeWithoutActionBar" />
+    <activity
+      android:name=".app.help.thirdparty.LicenseTextViewerActivity"
+      android:label="@string/faq_activity_title"
+      android:theme="@style/OppiaThemeWithoutActionBar" />
+    <activity
+      android:name=".app.help.thirdparty.ThirdPartyDependencyListActivity"
+      android:label="@string/third_party_dependency_list_activity_title"
+      android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.home.HomeActivity"
+      android:label="@string/license_text_viewer_activity_title"
```

Comment:
Please remove this label. This is incorrect.

Reply:
Done

Reply:
As the title will keep changing, what should be the correct label for this?

Reply:
This is `HomeActivity`. We are not showing any license here.

Reply:
Ah, sorry I didn't notice.
Removed it.